ic-http-outcall/README.md

```diff
@@ -0,0 +1,20 @@
+# HTTP Outcall Project
+=========================
+
+## Overview
+
+The HTTP Outcall project provides a set of Rust crates for making HTTP requests from canisters on the Internet Computer. The project includes three main crates: 
+- `ic-http-outcall-api` - common types.
+- `ic-http-outcall-proxy-canister` - canister to process pending HTTP outcalls.
```

Comment:
Do we another canister for this? Can this be a service to be embedded by other canisters instead? E.g. like ic-metrics for example?